File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
       int length = nums.length;
       int sum = (int) length*(length+1)/2;
        
        int arrSum = 0;
       for(int i=0;i<length;i++){
        arrSum+= nums[i];
       }

       return sum - arrSum;
    }
}

File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let length = nums.length;

    let originalSum = (length * (length + 1))/2;

    let ArrayElementSum = findSum(nums);

    function findSum(nums){
        let sum = 0;

        for(let i=0;i<nums.length;i++){
            sum = sum+nums[i];
        }

        return sum;
    }



    return originalSum - ArrayElementSum;
};